lib/workers/pr/body/index.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { platform } from '../../../../test/util';
+import { getConfig } from '../../../config/defaults';
+import type { RenovateConfig } from '../../../config/types';
+
+import { getPrBody } from '.';
+
+describe('workers/pr/body', () => {
+  let config: RenovateConfig;
+  beforeEach(() => {
```

Comment:
You'll probably want to mock `getPrConfigDescription` and `getControls` as these make `git` commands.

Reply:
Never did that before, can you confirm that I did the right things ?